src/main/java/org/spongepowered/api/text/serializer/TextSerializers.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * This file is part of SpongeAPI, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.api.text.serializer;
+
+import org.spongepowered.api.Sponge;
+import org.spongepowered.api.text.Text;
+import org.spongepowered.api.text.action.ClickAction;
+import org.spongepowered.api.text.action.HoverAction;
+
+/**
+ * The list of {@link TextSerializer} available together with SpongeAPI.
+ */
+public final class TextSerializers {
+
+    private TextSerializers() {
+    }
+
+    /**
+     * The {@link TextSerializer} for plain text input, without support for
+     * formatting or special text types.
+     */
+    public static final SafeTextSerializer PLAIN = null;
+
+    /**
+     * The {@link FormattingCodeTextSerializer} using the internal formatting
+     * char.
+     *
+     * <p><b>Note:</b> This serializer is ONLY available for compatibility
+     * reasons. Using texts with legacy formatting codes as-is, without using
+     * this {@link FormattingCodeTextSerializer} is broken in many ways and
+     * should not be used anymore unless really necessary. It is recommended to
+     * use a different text serializer such as {@link #FORMATTING_CODE}
+     * instead.</p>
+     *
+     * @see FormattingCodeTextSerializer
+     * @deprecated Legacy formatting codes are being phased out of Minecraft
+     */
+    @Deprecated
+    public static final FormattingCodeTextSerializer LEGACY = null;
```

Comment:
Eh, I'd rather keep it the way it is, it's used quite frequently in the implementation (and probably less in plugins) so it would be annoying having to type that every time.